src/store/auth/actions.ts

```diff
@@ -195,6 +195,30 @@ export function signUp(data) {
   }
 }
 
+export function validateEmail(data) { //checks if email is already registered
+  //somewhat hacky way to check, but apparently the best way currently
+  return dispatch => {
+    dispatch(loadingStart());
+    let email = data.email.toLowerCase().trim();
+    let dummyPass = "_";
+    Auth.confirmSignUp(email, dummyPass, {
+      forceAliasCreation: false
+    }).then(data => {
+      console.log(data);
```

Comment:
I think we can remove this `.then` function since we always expect this promise to fail?